conf/base/parameters.yml

```diff
@@ -79,4 +79,34 @@ gtr:
       param_requests: ${.._param_requests}
       label: projects
 
-test: False
\ No newline at end of file
+test: False
+
+lens:
+  data_collection:
+    api:
+      base_url: 'https://api.lens.org/patent/search'
+      token: QrN2bSOp8NiVSktDd8gWZ8Y5XMDsejM8hnID6dgUr1F80nSHvBfU # ${..lens_token}
```

Comment:
Is it ok for this token to be displayed publicly?

Reply:
There's no straightforward way of doing this - following your comment I've implemented a popular solution in a related issue, that includes an additional node that fetches credentials and passes as output the lens token. Not optimal, but seems to be the workaround.